ui/src/components/common/Holdings.tsx

```diff
@@ -207,36 +213,39 @@ const SplitForm: React.FC<SplitFormProps> = ({ deposit }) => {
             return
         }
 
-        if (splitAssetDecimal >= Number(asset.quantity)) {
-            const error = {
-                header: 'Invalid Split Quantity',
-                message: `The splitting quantity must be less than ${asset.quantity}`
-            };
-            throw error;
-        }
-
-        if (splitAssetDecimal <= 0 ){
-            const error = {
-                header: 'Invalid Split Quantity',
-                message: `The splitting quantity must be greater than 0.`
-            };
-            throw error;
-        }
-
         const args = { quantities: [String(splitAssetDecimal)] };
         const cid = deposit.contractId as ContractId<AssetDeposit>
         await ledger.exercise(AssetDeposit.AssetDeposit_Split, cid, args)
         setSplitAssetDecimal(undefined)
     }
 
+    const validateSplitNumber = (event: React.SyntheticEvent, result: any) => {
+        const number = Number(result.value)
+
+        if (number >= Number(asset.quantity)) {
+            return setSplitNumberError(`The splitting quantity must be less than ${asset.quantity}`)
+        }
+
+        if (number < 0) {
+            return setSplitNumberError(`The splitting quantity must be greater than 0.`)
```

Comment:
here the error message does not agree with the check. I think it should be `<=0`